### DISCUSSION TITLE
Combo burst sprites

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#30152] [Q&A]
*sorry if this is already answered somewhere*
Since combo bursts are coming to Lazer, i was wondering if the sprites for them will be redrawn or will they stay the same?
I could see the Pippi combo burst staying the same, but since Yuzu's combo burst uses his old design, it should be redrawn.
(I'm not asking necessarily if this is going to be already on the "release" of combo bursts, but if this is planned at all)

**Comment by peppy** [comment]
First we need to implement the combo bursts. I do believe we could do with some fresh artwork for them, though.
